Core/CoreTests/Use Cases/GetAssignmentsTests.swift

```diff
@@ -263,4 +263,86 @@ class GetAssignmentsTests: CoreTestCase {
         XCTAssertNotNil(assignment?.rubric)
         XCTAssertNotNil(assignment?.rubric?.first?.ratings?.first)
     }
+
+    func testItChangesRubrics() {
+        Assignment.make(["id": "2", "courseID": "2", "rubric": Set([Rubric.make(["id": "1"])])])
+
+        let apiAssignment = APIAssignment.make([
+            "id": "2",
+            "rubric": [APIRubric.fixture(["id": "2"])],
+            ])
+
+        let getAssignment = GetAssignment(courseID: "1", assignmentID: "2", include: [])
+        try! getAssignment.write(response: apiAssignment, urlResponse: nil, to: databaseClient)
+
+        let assignments: [Assignment] = databaseClient.fetch()
+        let assignment = assignments.first
+        XCTAssertNotNil(assignment)
+        XCTAssertNotNil(assignment?.rubric)
+        XCTAssertEqual(assignment?.rubric?.first?.id, "2")
+
+        //  make sure old existing rubrics were deleted
```

Comment:
Is this comment right? On 268 you make a rubric so it may or may not have been deleted, right?

Reply:
The id should change from 1 to 2.  So the assertions confirm that there is only 1 rubric with id 2